pkg/docker/index.html

```diff
@@ -152,7 +152,10 @@ <h1 translatable="yes">Can&rsquo;t connect to Docker</h1>
           </tr>
           <tr>
             <td translate>Image:</td>
-            <td colspan="3" id="container-details-image"></td>
+            <td colspan="3">
+            <div id="container-details-image"></div>
+            <div id="container-details-image-id"></div>
```

Comment:
If you are comparing across machines or registries having the id is pretty helpful. Right now ids are very prominent but fixing this bug ends up removing them from a lot of places. I felt that we should still display it somewhere.  But if you disagree I can remove it.